gym/f110_gym/envs/track/utils.py

```diff
@@ -24,7 +24,7 @@ def find_track_dir(track_name: str) -> pathlib.Path:
     FileNotFoundError
         if no map directory matching the track name is found
     """
-    map_dir = pathlib.Path(__file__).parent.parent.parent.parent / "maps"
+    map_dir = pathlib.Path(__file__).parent.parent.parent.parent.parent / "maps"
```

Comment:
Is there a reason we do this instead of importlib_resources?

Reply:
because:
1. just need the path, have to make maps/ into a package for importlib to work, that means at all levels with every single track.
2. code always runs on file system, not in zip or wheel